pennylane/ops/op_math/composite.py

```diff
@@ -112,13 +112,18 @@ def _op_symbol(self) -> str:
     @property
     def data(self):
         """Create data property"""
-        return [op.data for op in self]
+        data = []
+        for op in self:
+            data.extend(op.data)
+        return data
```

Comment:
```suggestion
                return [d for op in self for d in op.data]
```
This seems to be slightly more efficient.